proto/api.go

```diff
@@ -79,6 +79,7 @@ type Request interface {
 	gogoproto.Message
 	// Header returns the request header.
 	Header() *RequestHeader
+	KVHeader() *KVRequestHeader
```

Comment:
Structured would use Request/Response with KVHeader() returning nil. That was my plan.